src/sidebar/components/test/SidebarTabbedPanel-test.js

```diff
@@ -4,34 +4,40 @@ import {
 } from '@hypothesis/frontend-testing';
```

Comment:
The name of this file needs updating to match `SidebarTabs.tsx`.

Reply:
Ugh 😅